ecs-cli/modules/utils/compose/ecs_params_reader_test.go

```diff
@@ -77,6 +77,9 @@ task_definition:
   services:
     mysql:
       essential: false
+      cpu_shares: 3
+      memory: 10
```

Comment:
Are these realistic values for container `memory` and `memory_reservation`? I'm wondering if we want to provide full support for a memory string the way [libcompose](https://github.com/docker/libcompose/blob/master/yaml/types_yaml.go#L40) currently does (e.g. "1gb" etc).

Reply:
good point, we should apply the same logic here